Libplanet/Net/Swarm.cs

```diff
@@ -1803,14 +1794,11 @@ await foreach (Block<T> block in blocks)
 
                         cancellationToken.ThrowIfCancellationRequested();
 
-                        // As actions in this block should be rendered
-                        // after actions in stale blocks are unrendered,
-                        // given the `render: false` option here.
                         workspace.Append(
                             block,
                             DateTimeOffset.UtcNow,
                             evaluateActions: evaluateActions,
-                            renderActions: false
+                            renderActions: evaluateActions
```

Comment:
Is it okay we render before unrender? previous comments seemed to mention it... 😕

Reply:
You're right, I was mistaken about renders. I fixed it and added a test case for it.